### DISCUSSION TITLE
Gruppenzusammensetzung 30-30 maintenance und 30-30 maintenance 2

### DISCUSSION THREAD

**Discussion by marcbiehl** [irgroup-classrooms/dis08-2021#64] [Q&A]
Hallo Herr Schaer,

bei uns gab es Probleme mit der Gruppenzusammensetzung, können sie uns in eine Gruppe zusammenfügen?

Unsere Mitglieder sind:
Lia Heilmann
Cristina RIffarth
Benedikt Sellmaier
Marc Biehl

Danke im Vorraus

**Comment by phschaer** [comment]
Wenn wir mir kurz Ihre Github-Kennungen nennen, dann ja.

**Reply by phschaer** [reply]
Okay, habe nun alle Teilnehmer in die Gruppe "2.0" zusammengepackt. Das andere Team sollten Sie dann löschen. HTH.
